src/twitter/twitter_utils.bal

```diff
@@ -41,29 +42,37 @@ function constructRequestHeaders(http:Request request, string httpMethod, string
         string consumerSecret, string accessToken, string accessTokenSecret, string paramStr) returns error? {
     string serviceEndpoint = "https://api.twitter.com" + serviceEP;
     string paramString = paramStr.substring(0, paramStr.length() - 1);
-    string encodedServiceEPValue = check http:encode(serviceEndpoint, "UTF-8");
-    string encodedParamStrValue = check http:encode(paramString, "UTF-8");
-    string encodedConsumerSecretValue = check http:encode(consumerSecret, "UTF-8");
-    string encodedAccessTokenSecretValue = check http:encode(accessTokenSecret, "UTF-8");
+    string encodedServiceEPValue = check encoding:encodeUriComponent(serviceEndpoint, "UTF-8");
+    string encodedParamStrValue = check encoding:encodeUriComponent(paramString, "UTF-8");
+    string encodedConsumerSecretValue = check encoding:encodeUriComponent(consumerSecret, "UTF-8");
+    string encodedAccessTokenSecretValue = check encoding:encodeUriComponent(accessTokenSecret, "UTF-8");
 
     string baseString = httpMethod + "&" + encodedServiceEPValue + "&" + encodedParamStrValue;
-    byte[] baseStringByte = baseString.toByteArray("UTF-8");
+    byte[] baseStringByte = internal:toByteArray(baseString, "UTF-8");
```

Comment:
@wggihan Do we still use internal module?

Reply:
ballerina/internal module will be removed. This will now support through builtin. Simply use baseString.toBytes();